MIGRATION.md

```diff
@@ -0,0 +1,94 @@
+# Migration
+
+## Migrating from version 7 to 8
+
+As of version 8 `@fastify/swagger` is only responsible for generating valid
+swagger/openapi-specifications. The new `@fastify/swagger-ui` plugin is
+responsible for serving the swagger-ui frontend.
+
+Options in version 7 of `@fastify/swagger` related to the configuration
+of the swagger-ui frontend are now options of `@fastify/swagger-ui`.
+
+The `exposeRoute` option is removed.
+
+Following are the `@fastify/swagger-ui` options:
+
+| Option             | Default          | Description                                                                                                               |
+| ------------------ | ---------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| baseDir            | undefined        | Specify the directory where all spec files that are included in the main one using $ref will be located. By default, this is the directory where the main spec file is located. Provided value should be an absolute path without trailing slash.     |
```

Comment:
I might not be reading this correctly. Are we loading files from disk now? Hard no.

Reply:
The plugin already does this in static mode doesn't it?

Reply:
@mcollina 
No functionally is changed. As @Fdawgs explains this is just for the necessety to keep the behaviour same. 

In current @fastify/swagger in static mode the specification.baseDir has two meanings. One is to load referenced specification files based on that baseDir AND exposing the files with fastify-static.

In current @fastify/swagger when using static mode and setting specification.baseDir it results in exposing those referenced schema files also see: 
https://github.com/fastify/fastify-swagger/blob/d8a57a47ebb800d493c734dc8c662b27be09acae/lib/routes.js#L124

That part is also transplanted to fastify-swagger-ui
https://github.com/fastify/fastify-swagger-ui/blob/e85c5fc67cfa38ea76cb6e32237b7bb4294ab5a9/lib/routes.js#L124

So to handle it properly after the split, you need to set in @fastify/swagger baseDir and once in @fastify/swagger-ui. In @fastify/swagger baseDir will be used to load referenced specification files and in @fastify/swagger-ui it will used to expose files with fastify-swagger.

As I said: I did not change any thing significantly regarding the functionality, despite the ripping apart of swagger and swagger-ui

Reply:
The migration scripts does not mention this option being specific for static mode.

Reply:
```suggestion
| baseDir            | undefined        | Only relevant if `@fastify/swagger` used in static-mode and additional schema-files contain referenced schemas. Specify the directory where all spec files that are included in the main one using $ref will be located. By default, this is the directory where the main spec file is located. Provided value should be an absolute path without trailing slash.     |
```